woocommerce/woocommerce-gutenberg-products-block
# Checkout i2: Remove Sample Block.

We built Sample block to test various features in Checkout i2, it served it purpose and we're satisfied with his care and can let him go now.

I also created an issue to track the doc status of creating a block. #4840 

Closes #4838

### Testing
How to test the changes in this Pull Request:

1. Insert Checkout block
2. Make sure you can't insert sample block.
